src/content/blog/2023-12-29-eslint-v9.0.0-alpha.0-released.md

```diff
@@ -3,21 +3,78 @@ layout: post
 title: ESLint v9.0.0-alpha.0 released
 teaser: "We just pushed ESLint v9.0.0-alpha.0, which is a major release upgrade of ESLint. This release adds some new features and fixes several bugs found in the previous release. This release also has some breaking changes, so please read the following closely."
 image: release-notes-major.png
-draft: true
 authors:
-  - eslintbot
+  - mdjermanovic
+  - nzakas
 categories:
   - Release Notes
 tags:
   - Release
 ---
 
+## Highlights
 
+This version of ESLint is not ready for production use and is provided to gather feedback from the community before releasing the final version. Please let us know if you having any problems or feedback by creating issues on our [GitHub repo](https://github.com/eslint/eslint).
 
+Most of the highlights of this release are breaking changes, and are discussed further in the [migration guide](/docs/next/use/migrate-to-9.0.0). There are summaries of the significant changes below. (Less significant changes are included in the migration guide.)
 
-## Highlights
+### Node.js < v18.18.0, v19 no longer supported
+
+As of this post, Node.js v20.x is the LTS release, and as such we are [dropping support](https://github.com/eslint/eslint/issues/17595) for all versions of Node.js prior to v18.18.0 as well as v19.x.
+
+### Flat config now the default and has some changes
+
+Flat config is now the default configuration format for ESLint and eslintrc is officially deprecated. beginning in v9.0.0 and eslintrc will be deprecated but will still work. To continue using a eslintrc configuration file, you'll need to set the `ESLINT_USE_FLAT_CONFIG` environment variable to `false`.
```

Comment:
`beginning in v9.0.0 and eslintrc will be deprecated but will still work.` looks like a copy-paste error.

Reply:
Good catch